src/OpenTelemetry.Instrumentation.Runtime/description.md

```diff
@@ -0,0 +1,175 @@
+# Runtime metrics description
```

Comment:
Maybe "Runtime metrics overview" or "Runtime metric details" here instead of "description"? Also we should link to this from README.